### ISSUE TITLE
Can't run bash again if "shell integration" is enabled

### ISSUE THREAD

**Issue by Toub** [sedwards2009/extraterm#300]
I just installed Extraterm v0.55.0 with [shelll integration](https://github.com/sedwards2009/extraterm/blob/master/docs/guide.md#shell-integration) to work with bash.

I added the following line to `~/.bashrc`:

```bash
. $USER/.extraterm/extraterm-commands-0.55.0/setup_extraterm_bash.sh
```

Then, when opening extraterm, this message is displayed:

```bash
Setting up Extraterm support.
```

However, if I open bash again, it try to configure extraterm shell again, and __freezes__:

```bash
bash
Setting up Extraterm support. 
```

What about adding an environment variable in the install script to mark the setup as already completed and skip it, e.g.:

```
export EXTRATERM_COMMANDS_VERSION=0.55.0
```

Possibly related to https://github.com/sedwards2009/extraterm/issues/237

**Comment by Toub** [comment]
Hmm, I guess configuring extraterm in `~/.bashrc` is a bad idea.

Is there a better way to make it run at extraterm startup?

**Comment by sedwards2009** [comment]
Easily reproducible here.

**Comment by sedwards2009** [comment]
Fixed in v0.55.1.
